src/client/src/orangehrmRecruitmentPlugin/pages/AddJobVacancy.vue

```diff
@@ -141,7 +141,7 @@ const vacancyModel = {
   jobTitle: null,
   name: '',
   hiringManager: null,
-  numOfPositions: null,
+  numOfPositions: '',
```

Comment:
let's make this 0

Reply:
done

Reply:
Will discuss further this with kavinda since empty string is the 4x behaviour